internal/apidiff/apidiff.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const rootMod = "cloud.google.com/go"
+
+var repoMetadataPath string
+var verbose bool
+
+var root string
+var manifest map[string]manifestEntry
+var diffs map[string]string
+var errs map[string]error
+
+func init() {
+	flag.StringVar(&repoMetadataPath, "repo-metadata", "", "path to a repo-metadata-full JSON blob [required]")
+	flag.BoolVar(&verbose, "verbose", false, "enable verbose command logging")
+
+	root, _ = os.Getwd()
```

Comment:
If this was done outside of init, see above, I think we could check the error.

Reply:
Done.